modules/nf-core/bcftools/annotate/main.nf

```diff
@@ -10,6 +10,7 @@ process BCFTOOLS_ANNOTATE {
     input:
     tuple val(meta), path(input), path(index), path(annotations), path(annotations_index)
     path(header_lines)
+    path(rename_chrs)
```

Comment:
Would it make sense to also pass a meta that contains the info what renaming is done? (so from ucsc to ensembl ot the other way and so on?

Reply:
We could add a meta, I left it out for consistency with `header_lines`.

Reply:
I am fine with either. Then lets keep it like that for now.